api/src/test/java/org/openmrs/module/dhisreport/api/dhis/HttpDhis2ServerTest.java

```diff
@@ -0,0 +1,78 @@
+package org.openmrs.module.dhisreport.api.dhis;
```

Comment:
@maurya 
This test is for the Package `org.openmrs.module.dhisreport.api.dhis`, I have added this test in `org.openmrs.module.dhisreport.api.impl` branch because this test needed the config files which are used in this package.